tests/discovery/test_discovery.py

```
@@ -40,19 +78,41 @@ def wrap_signpost_uri(uri: str) -> str:
-ALL_CASES = DIRECT_CASES + SIGNPOSTED_CASES
+ALL_CASES = REMOTE_CASES + SIGNPOSTED_CASES
+
+
+class CustomHTTPRequestHandler(SimpleHTTPRequestHandler):
+    def guess_type(self, path):
+        ext = os.path.splitext(path)[1]
+        return HTTPD_EXTENSION_MAP.get(ext, super().guess_type(path))
+
+
+@pytest.fixture(scope="module", autouse=True)
+def http_server():
```

Ruff

Fixture `http_server` does not return anything, add leading underscore

(PT004)